src/tooltip.tsx

```diff
@@ -0,0 +1,151 @@
+import isString from 'lodash/isString';
+import merge from 'lodash/merge';
+import * as React from 'react';
+import styled from 'styled-components';
+import { TooltipPlacement } from './common-types';
+import RCTooltip from 'rc-tooltip';
+
+// The time in ms to show a tooltip after being triggered by a click event
+const CLICK_TIMEOUT = 1000;
+
+const Arrow = styled.div`
+	position: absolute;
+	width: 0;
+	height: 0;
+	border-color: transparent;
+	border-style: solid;
+
+	.rc-tooltip-placement-topLeft &,
+	.rc-tooltip-placement-topRight &,
+	.rc-tooltip-placement-top & {
+		bottom: 0;
+		left: 50%;
+		margin-left: -5px;
+		border-width: 5px 5px 0;
+		border-top-color: #000;
+	}
+
+	.rc-tooltip-placement-right & {
+		top: 50%;
+		left: 0;
+		margin-top: -5px;
+		border-width: 5px 5px 5px 0;
+		border-right-color: #000;
+	}
+
+	.rc-tooltip-placement-left & {
+		top: 50%;
+		right: 0;
+		margin-top: -5px;
+		border-width: 5px 0 5px 5px;
+		border-left-color: #000;
+	}
+
+	.rc-tooltip-placement-bottomLeft &,
+	.rc-tooltip-placement-bottomRight &,
+	.rc-tooltip-placement-bottom & {
+		top: 0;
+		left: 50%;
+		margin-left: -5px;
+		border-width: 0 5px 5px;
+		border-bottom-color: #000;
+	}
+`;
+
+const TooltipElementInner = styled.div`
+	background: black;
+	border-radius: 4px;
+	padding: 3px 8px;
+	text-align: center;
+`;
+
+interface TooltipComponentProps {
+	tooltip:
+		| string
+		| {
+				text: string;
+				trigger?: 'hover' | 'click';
+				placement?: TooltipPlacement;
+				containerStyle?: React.CSSProperties;
+				innerStyle?: React.CSSProperties;
+				arrowStyle?: React.CSSProperties;
+		  };
+	children: React.ReactElement;
+}
+
+const overlayStyles = {
```

Comment:
Maybe not all of these are necessary? We don't have to include everything from the previous tooltip, we just need to make the new one work 😄

Reply:
Umm Sure, will remove which seems un-ncessary